.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -18,12 +18,16 @@ on:
           - eltwise.gelu
           - eltwise.cos 
           - eltwise.sin                          
+          - eltwise.clamp.clamp_interleaved
```

Comment:
File names need to match these list options, pls rename the file structure to match, or edit this list.